specification/marketplacecatalog/resource-manager/Products/preview/2022-09-01-preview/examples/Operations_Get.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Marketplace/products",
```

Comment:
@lirenhe I am a little confused as to the correct naming conventions for operations controllers.
The documentation says that naming should be done "as per RBAC" but I'm not entirely sure what this means.